qsirecon/data/pipelines/TORTOISE.yaml

```diff
@@ -0,0 +1,19 @@
+anatomical: []
```

Comment:
I'd love to take full advantage of the YAML format and add in comments, but that's something someone with actual knowledge should do.

Reply:
Do you mean like for describing what the parameters do?

Reply:
I was more thinking about top-level descriptions of the pipelines (e.g., where it comes from, when you'd want to use it), but informal descriptions of the individual parameters would be great too.

Reply:
do we want it in the yaml or the online docs or the docstring? We could always put a link to the online docs for it in the comments. Then those docs would either come from the docs directory or the docstring

Reply:
Either works for me